project_event/views/project_task_view.xml

```diff
@@ -196,6 +196,7 @@
                                    domain="[('activity_task_type','=','activity')]" attrs="{'readonly': [('task_state', 'in', ['done','accepted'])]}"/>
                             <field name="category_id" required="1"  attrs="{'readonly': [('task_state', 'in', ['done','accepted'])]}"/>
                             <field name="color" widget="color" readonly="1" attrs="{'readonly': [('task_state', 'in', ['done','accepted'])]}"/>
+                            <field name="font_color" widget="radio" readonly="1" attrs="{'readonly': [('task_state', 'in', ['done','accepted'])]}"/>
```

Comment:
no need to display this field, put it invisible. remove readonly=1 and attrs=readonly

Reply:
@rimbendhaou  ok